### PR TITLE
LPD-18700 [Marketplace] Fix image order, and image size on apps search page

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/components/OrderDetailsHeader.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ const OrderDetailsHeader: React.FC<OrderDetailsProps> = ({
 			<img
 				alt="App Icon"
 				className="order-details-publisher-icon"
-				src={showAppImage(image)}
+				src={(image)}
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/components/OrderDetailsHeader.tsx:39]:
```suggestion
				src={image}
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/StorefrontPage/CustomizeAppStorefrontPage.tsx [hunk 2]
```diff
@@ -252,7 +252,7 @@ export function CustomizeAppStorefrontPage({
 								appStorefrontImages;
 							},
 							file: image.file,
-							index,
+							index:index+1,
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/StorefrontPage/CustomizeAppStorefrontPage.tsx:255]:
```suggestion
							index: index + 1,
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
Please

**Comment by kevenleone** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:forward

**Pull request closed without merging**